.github/workflows/build-packages.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Install packages
         run: |
           apt-get update && apt-get install git cmake wget build-essential pkg-config libssl-dev clang g++ -y --option=Dpkg::Options::=--force-confdef &&
-          wget -q https://static.rust-lang.org/dist/rust-1.84.1-x86_64-unknown-linux-gnu.tar.gz -O /tmp/rust.tar.gz &&
```

Comment:
You may use `actions-rust-lang` for setup: [actions-rust-lang/setup-rust-toolchain](https://github.com/actions-rust-lang/setup-rust-toolchain)

Reply:
I would advocate for action-based solution as it has the profits of:
* caching toolchain;
* respecting `rust-toolchain.toml` (it is always easier to have less sources of truth, e.g. version number);
* easier to maintain.

But I don't insist, just showing some pros

Reply:
this happens once with a tag.

Reply:
I explain once again that build should be clean, without the involvement of left-wing, vulnerability-prone dependencies.
secondly, the effectiveness of this is questionable, because it happens during the build tag.